docs/contributing-guide/6. submitting-a-pr.md

```diff
@@ -50,22 +52,22 @@ rush changelog
 
 This will start an interactive tool that will:
 
-1. First ask you to select whether your change is a Major, Minor, or Patch (for more information on which to choose see: [semver docs](https://semver.org/))
+1. First ask you to select whether your change is a Major, Minor, or Patch (for more information on which to choose see [Semantic Versioning](https://semver.org/)).
 
 1. Then ask for a short description of your change - this is what will be included in the changelog when a new version is released, so keep it concise and meaningful. See [Tips for writing meaningful changelog entries](../reviewer-notes/tips-for-writing-changelog-entries).
 
-### Generating new snapshot for your UI changes
+### Generating new snapshots for your UI changes
 
-If you are making an ui change in your PR, you might invalidate some screen snapshot in our automation visual tests.
+If you are making a UI change in your PR, you might invalidate some screen snapshots in our automated visual tests.
 
-1. Check the failing github actions to see if that is a desired UI change
+1. Check the failing Github Actions to see if the failure is due to a UI change.
 
    ![screenshot showing downloadable snapshot artifacts in a failed github action](../images/check-snapshots.png)
-1. Add `update_snapshot` label to the PR
-1. Wait github actions to generate new snapshot for your PR
-1. Validation pipeline will re-run after new snapshot gets generated
+1. Add `update_snapshot` label to the PR.
+1. Wait for Github Actions to generate new snapshots for the PR.
+1. The validation pipelines will be re-run after new snapshots are generated.
```

Comment:
```suggestion
1. The validation pipelines will re-run after new snapshots are generated.
```

Reply:
I think the use of "be" is necessary in this context to indicate the future action of the validation pipelines being re-run